host/sgx/sgxload.c

```diff
@@ -361,13 +364,33 @@ oe_result_t oe_sgx_create_enclave(
         // Wrong code path
         result = OE_UNSUPPORTED;
         goto done;
-#endif // OEHOSTMR
+#endif // !defined(OEHOSTMR)
     }
 
     /* Create SECS structure */
     if (!(secs = _new_secs((uint64_t)base, enclave_size, context)))
         OE_RAISE(OE_OUT_OF_MEMORY);
 
+    /*
+     * Load desired enclave start address. NOTE: Currently, this value is NULL
+     * when zero base enclave is not enabled. Also, base_address has to be
+     * aligned to OE_PAGE_SIZE.
+     */
+    base_address = (void*)context->start_address;
+    if ((uint64_t)base_address % OE_PAGE_SIZE)
+        OE_RAISE(OE_INVALID_PARAMETER);
+
+    if (context->create_zero_base_enclave)
+    {
+        enclave_elrange.enclave_image_address = (uint64_t)base_address;
```

Comment:
It would be good to add comments explain the three settings and their relations.